editorconfig.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/blang/semver"
```

Comment:
It'd be nice if we could do this without import this library. Maybe not easy, though.

Reply:
Do you have another one in mind?

Reply:
Sorry, I meant: it's be nice if we could do this without importing *any* library at all.

But as I said, it may be not easy to do so. If you think it's not worth the effort trying to parse the version ourselves, feel free to proceed and merge this PR as is. 😉

Reply:
I'll see how much is needed.

> _It can be better to copy a little code than to pull in a big library for one function. Dependency hygiene trumps code reuse._
>
> Rob Pike

Reply:
I've played with that and didn't manage to have something anywhere as nice.